src/Framework/Shims/current_datetime.php

```diff
@@ -10,12 +10,11 @@
     /**
      * Retrieves the current time as an object using the site’s timezone.
      *
-     * @return DateTimeImmutable
-     * @throws Exception
+     * @return DateTimeImmutable|false
      */
     function current_datetime()
     {
-        return new DateTimeImmutable('now', wp_timezone());
```

Comment:
@jonwaldstein I had lifted this straight from Trac. Is there a particular reason to use `date_create_immutable` instead?

Reply:
it's the same thing but avoids having to add unnecessary exception docs for _everything_ that uses it

Reply:
Ah, I see now.

> I just changed to date_create_immutable to avoid exception doc hell
https://github.com/impress-org/givewp/pull/6298#discussion_r824827751